use_cases.md

```diff
@@ -752,10 +750,10 @@ according to the CaaS platform requirements:
 The Capsule controller, watching for Namespace creation,
 creates the Network Policies for each Namespace in the tenant.
 
-The tenat user (e.g. Joe) has access these network policies:
+The tenat user (e.g. Joe) has to access these network policies:
```

Comment:
It means the tenant user has the administrative rights to view and list _NetworkPolicies_, maybe consider to change the verb.

Reply:
@YashThakare please, fix this.